fvtest/porttest/omrsockTest.cpp

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "omrport.h"
+#include "omrportsock.h"
+#include "testHelpers.hpp"
+
+/**
+ * Start a server which includes create a socket, bind to an address/port and 
+ * listen for clients.
+ * 
+ * @param[in] portLibrary
+ * @param[in] testname The name of the calling test.
+ * @param[in] addrStr The address of the server.
+ * @param[in] port The server port.
+ * @param[in] family Socket address family wanted.
+ * @param[out] serverSocket A pointer to the server socket. 
+ * @param[out] serverAddr The socket address of the server created.
+ * 
+ * @return 0 on success, return an error otherwise.
+ */ 
+int32_t
+startServer(struct OMRPortLibrary *portLibrary, const char *testName, const char *addrStr, const char *port, int32_t family, omrsock_socket_t *serverSocket, omrsock_sockaddr_t serverAddr) 
```

Comment:
Use one convention. Format of test names: `sock_test0`.
```suggestion
start_server(struct OMRPortLibrary *portLibrary, const char *testName, const char *addrStr, const char *port, int32_t family, omrsock_socket_t *serverSocket, omrsock_sockaddr_t serverAddr) 
```